sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -120,7 +159,26 @@ def _renewable(self, renewable):
             return False
         return True
 
-    def _auto_lock_renew(
+    def _dispatch_worker(self):
+        self._last_activity_timestamp = time.time()
+        while not self._shutdown.is_set() and self._running.is_set():
```

Comment:
What's the distinction between the 2 flags here?

Reply:
the first shutdown is for the whole life cycle control of the renewer, once shut down is set (close called or exit the context manager), it could no longer be used.

the second is for controling dispatch thread

Reply:
Gotcha - maybe we could use a more descriptive name for the second? e.g. `self._running_dispatcher`

Reply:
name updated